Tests/MapboxMapsTests/Annotations/Generated/CircleAnnotationIntegrationTests.swift

```diff
@@ -42,7 +42,7 @@ final class CircleAnnotationIntegrationTests: MapViewIntegrationTestCase {
         manager.annotations.append(annotation)
 
         expectation(for: NSPredicate(block: { (_, _) in
-            guard let layer: CircleLayer = try? self.style?.layer(withId: self.manager.layerId) else {
+            guard let layer: CircleLayer = try? self.style?.layer(withId: self.manager.layerId, type: CircleLayer.self) else {
```

Comment:
Here you can remove the `: CircleLayer` due to this change. Please apply this throughout this PR.